diff -up LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/librecad/src/src.pro.install LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/librecad/src/src.pro
--- LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/librecad/src/src.pro.install	2013-10-18 01:06:00.805457502 +0100
+++ LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/librecad/src/src.pro	2013-10-18 01:06:00.807457490 +0100
@@ -57,7 +57,6 @@ unix {
         DEFINES += QC_APPDIR="\"librecad\""
         DEFINES += QINITIMAGES_LIBRECAD="qInitImages_librecad"
         RC_FILE = ../res/main/librecad.icns
-        QMAKE_POST_LINK = cd $$_PRO_FILE_PWD_/../.. && scripts/postprocess-unix.sh
     }
 }
 win32 {
diff -up LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/scripts/postprocess-unix.sh.install LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/scripts/postprocess-unix.sh
--- LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/scripts/postprocess-unix.sh.install	2013-10-18 01:06:00.807457490 +0100
+++ LibreCAD-f87828ec55def73aaa8c3c3e158fc1abc8171b63/scripts/postprocess-unix.sh	2013-10-18 01:06:38.471238579 +0100
@@ -3,7 +3,7 @@
 THISDIR="`pwd`"
 LCDIR="${THISDIR}/librecad"
 PIDIR="${THISDIR}/plugins"
-RESOURCEDIR="${THISDIR}/unix/resources"
+RESOURCEDIR=$BUILDDIR
 APPDATADIR="${THISDIR}/unix/appdata"
 TSDIRLC="${LCDIR}/ts"
 TSDIRPI="${PIDIR}/ts"
