Index: librecad-1.0.0/desktop/librecad.desktop
===================================================================
--- librecad-1.0.0.orig/desktop/librecad.desktop
+++ librecad-1.0.0/desktop/librecad.desktop
@@ -4,8 +4,9 @@ Type=Application
 Name=LibreCAD
 Comment=A professional CAD System
 Comment[es]=Un sistema CAD profesional
+Comment[ru]=Система автоматизированного проектирования
 TryExec=librecad
 Exec=librecad %F
 Icon=librecad
 Categories=Graphics;Engineering;
-MimeType=image/vnd.dxf
+MimeType=image/vnd.dxf;
